### ISSUE TITLE
Developers don't seem to use bootstrap generated events with modals

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/css_layout_with_bootstrap#9]
Which I believe led to many of their problems using them.

**Comment by jrhorn424** [comment]
Like `modalClosed` etc?

**Comment by gaand** [comment]
Yeah.  It may actually be more complicated than my issue indicates.  I need to research it a bit.

**Comment by MicFin** [comment]
Any additions to this issue for cohort 016?  Planning to present without closing the issue.

**Comment by gaand** [comment]
@ga-wdi-boston/core Please ping me when preparing this talk next time so we can discuss this (unless it's made obsolete buy other curriculum changes).

**Comment by jrhorn424** [comment]
Is it appropriate to address this during the Modals lab?

**Comment by jrhorn424** [comment]
Related #28

**Comment by jrhorn424** [comment]
This has come up a few times. Most recently, folks are using `setTimeout` to indeterminately avoid a race condition with handlebars-rendered modals.

@raq929 recalls this happening many times in 015 as well (though no one used setTimeout to avoid it).

My preferred solution is to use the `modal` instance methods bootstrap provides on modal elements via its jQuery plugins. http://getbootstrap.com/javascript/#modals-methods

**Comment by raq929** [comment]
I'd prefer to get rid of using modals altogether and teach bootstrap generated events a different way.

**Comment by jrhorn424** [comment]
> I'd prefer to get rid of using modals altogether and teach bootstrap generated events a different way.

Me too! #28

**Comment by Jcornmanhomonoff** [comment]
Including explicit discussion about not using jQuery show/hide & using Bootstrap methods instead.
